apps/src/aiTutor/views/UserMessage.tsx

```diff
@@ -12,50 +12,47 @@ interface UserMessageProps {
   message: ChatCompletionMessage;
 }
 
-const INAPPROPRIATE_MESSAGE = 'This chat is inappropriate.';
-const TOO_PERSONAL_MESSAGE = 'This chat is too personal.';
-const ERROR_MESSAGE =
+const PROFANITY_VIOLATION_USER_MESSAGE =
+  'This chat has been hidden because it is inappropriate.';
+const PII_VIOLATION_USER_MESSAGE =
+  'This chat has been hidden because it contains personal information.';
+const ERROR_USER_MESSAGE =
```

Comment:
I standardized the variable names in this file so they all refer to "PII violation" and "profanity violation" instead of the more generic "inappropriate"/"personal."